src/cli/java/org/commcare/util/cli/ApplicationHost.java

```diff
@@ -403,9 +411,34 @@ private void initUser() {
         System.out.println("Setting logged in user to: " + u.getUsername());
     }
 
-    public static void restoreUserToSandbox(UserSandbox sandbox, String username, final String password) {
+    public void restoreUserToSandbox(UserSandbox sandbox, String username,

```

Comment:
This needs to remain static because it is used by the clojure commcare CLI. I'm updating it now because I'm also extending the clojure cli to include sync functionality.